amethyst_network/src/systems/network_client.rs

```diff
@@ -0,0 +1,50 @@
+extern crate ron;
+
+use specs::{Entities, Entity, Join, System, WriteStorage};
+use std::net::UdpSocket;
+use std::net::IpAddr;
+use std::str;
+
+use amethyst_core::transform::*;
+
+use ::components::netsync::*;
+
+pub struct NetClientSystem {
+    pub socket:UdpSocket,
+}
+
+impl NetClientSystem {
+    pub fn new()->NetClientSystem{
+        println!("Starting client socket");
```

Comment:
Gotta finish my logging PR :)

Reply:
can you post the status of the logging PR as a new comment please @torkleyy ?